.gitignore

```diff
@@ -41,7 +40,12 @@ build/
 /tools/permuter_settings.toml
 
 # Assets
-/assets/*
```

Comment:
This is to make it so people hopefully won't have to go modifying gitignore when adding new asset directories